jenkins/bootstrap_test.py

```diff
@@ -2199,7 +2203,8 @@ def test_valid_job_envs(self):
                     if (
                             '--env-file=jobs/platform/gce.env' not in args
                             and '--env-file=jobs/platform/gke.env' not in args
-                            and '--check-leaked-resources=true' in args):
+                            and '--check-leaked-resources=true' in args
+                            and 'generated' not in config[job].get('tags', [])):
```

Comment:
Do we expect to enforce all generated jobs (even after we migrate more jobs) to have `--check-leaked-resources=true`?